drawBot/context/imageContext.py

```diff
@@ -35,13 +49,18 @@ def _writeDataToFile(self, data, path, multipage):
             firstPage = pageCount - 1
             pathAdd = ""
         outputPaths = []
+        imageResolution = options.get("imageResolution", 72.0)
+        properties = {}
+        for dbKey, nsKey in _nsImageOptions.items():
+            if dbKey in options:
+                properties[nsKey] = options[dbKey]
```

Comment:
there are some converters needed: if a nscolor is required it would be nice just to provide `(1, 0, 0)`

Reply:
Yes, but that seems to be the only one. I'll fix that.

Reply:
and maybe the NSData keys, drawBot would like to provide raw data, not an NSData object, actually I dont know what kind of data should be provided.